Exercises/data-types-overview/data-types-overview.js

```diff
@@ -0,0 +1,170 @@
+
+
+// Declare a variable on each type using two options: plain creation, creation with constructor.
+
+//number
+let number = 123;
+console.log(number);
+
+//string
+let string = "some text";
+console.log(string);
+
+//boolean
+let bool = true;
+console.log(bool);
+
+//undefined
+let undef = undefined;
+console.log(undef);
+
+//null
+let nul = null;
+console.log(nul);
+
+//object
+let object = {
+    a: 1,
+    b: "text",
+    c: {d: "a", e: null}
+}
+console.log(object);
+
+object2 = new Object()
+console.log(object2);
+
+
+// Use operator typeof on each of them.
+
+//number
+let number = 123;
+console.log(typeof(number));
+number2 = new Number(777.123);
+console.log(typeof(number2));
+
+
+//string
+let string = "some text";
+console.log(typeof(string));
+string2 = new String("new text");
+console.log(typeof(string2));
+
+//boolean
+let bool = true;
+console.log(typeof(bool));
+
+//undefined
+let undef = undefined;
+console.log(typeof(undef));
+
+//null
+let nul = null;
+console.log(typeof(nul));
+
+//object
+let object = {
+    a: 1,
+    b: "text",
+    c: {d: "a", e: null}
+}
+console.log(typeof(object))
+
+// Demonstrate object's references.
+
+let object = {
```

Comment:
it task meant to demonstrate that variable is actually reference to object in memory, and how it affects on you